ods_ci/tests/Tests/400__ods_dashboard/415__ods_dashboard_projects/415__ods_dashboard_projects_permissions_mgmt.robot

```diff
@@ -248,7 +248,7 @@ ${username} Should Have Edit Access To ${project_title}
     Open Data Science Projects Home Page
     Reload Page If Project ${project_title} Is Not Listed
     Wait Until Project Is Listed    project_title=${project_title}
-    Open Data Science Project Details Page    ${project_title}    tab_id=permissions
+    Open Data Science Project Details Page    ${project_title}
     Permissions Tab Should Not Be Accessible
```

Comment:
I don't know if this got updated after the UI changes, but it might be outdated (passing because the searched element doesn't exist anymore)

Reply:
https://github.com/red-hat-data-services/ods-ci/pull/1333/files 
It was done here

Reply:
it was done for `Open Data Science Project Details Page` keyword. Are you sure `Permissions Tab Should Not Be Accessible` and `Permissions Tab Should Be Accessible` got reviewed?